src/presenter/trip-event-presenter.js

```diff
@@ -1,67 +1,60 @@
-import { render, replace } from '../framework/render.js';
-import ListEvent from '../view/event-list-view.js';
-import EventsItemView from '../view/events-item-view.js';
-import EventEdit from '../view/event-edit.js';
+import { render } from '../framework/render.js';
 import ListEmpty from '../view/list-empty.js';
-export default class EventPresenter {
+import PointPresenter from './point-presenter.js';
+import EventsListView from '../view/events-list-view.js';
 
-  #eventComponent = new ListEvent();
+export default class EventPresenter {
```

Comment:
В идеале нужно ещё познакомиться с комментариями JSDoc Для документирования кода, когда есть неочевидные переменные и нужно помнить что внутри, например так можно указать, что points содержит Map со значениями строка/ презентер -- это позволит ИДЕ/редактору автодополняеть код при работе с такими переменными
```suggestion
export default class EventPresenter {
  #eventsListView = new EventsListView();
  #listContainer = null;

  /**
   * Список всех презентеров точек
   * @type {Map<string, PointPresenter>}
   */
  #points = null;

  #pointsModel = null;
  #offersModel = null;
  #destinationsModel = null;
```